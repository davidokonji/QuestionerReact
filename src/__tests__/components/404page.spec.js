import React from 'react';
import { shallow } from 'enzyme';
import Notfound from '../../components/404page';

describe('404 page', () => {
  test('should render 404 page correctly', () => {
    const wrapper = shallow(<Notfound />);

    expect(wrapper).toMatchSnapshot();
  });
});
