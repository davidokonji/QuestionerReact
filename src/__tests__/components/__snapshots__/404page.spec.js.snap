// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 page should render 404 page correctly 1`] = `
<Fragment>
  <Header
    navClass="nav_header"
    page="landingpage"
  />
  <div
    id="notfound"
  >
    <div
      className="notfound"
    >
      <div
        className="notfound-404"
      >
        <h3>
          Oops!
           Page 
          not found
        </h3>
        <h1>
          <span>
            4
          </span>
          <span>
            0
          </span>
          <span>
            4
          </span>
        </h1>
      </div>
      <h2>
        we are sorry, page you requested was not found
      </h2>
      <Link
        className="homeLink"
        to="/"
      >
        Home
      </Link>
    </div>
  </div>
</Fragment>
`;
